lib/parlour/rbi_generator/namespace.rb

```diff
@@ -106,18 +106,24 @@ def constants
         )
       end
 
-      sig { params(object: T.untyped, block: T.proc.params(x: Namespace).void).void }
-      # Given a Class or Module object, generates all classes and modules in the
+      sig { params(constant: Module, block: T.proc.params(x: Namespace).void).void }
+      # Given a Module object, generates all classes and modules in the
```

Comment:
Good catch, let's also update the comment in that case

Reply:
I'm not sure how but `constant` should appear somewhere in the sentence to make the link with `Module object`

Reply:
I can reword it to: "Given a constant (i.e. a Module instance), generates ..."

How does that sound?

Reply:
Yeah I understand the comment better, thanks! 👍